### ISSUE TITLE
libjvm.so does not exist or is not executable

### ISSUE THREAD

**Issue by alblue** [iovisor/bpftrace#1102]
When using `bpftrace` to track huge object allocations in a JVM process (as per https://twitter.com/alblue/status/1220290172429926402) an error message is given if the library is not executable:

```
ls -l libjvm.so 
-rw-r--r-- 1 root root 22919740 Dec 27 05:17 libjvm.so
```

If I make it executable, the problem goes away. But it should not be necessary to make a library executable in order to make this work; it looks like a spurious check in the `bpftrace` code, that is looking for executables.

**Comment by fbs** [comment]
Which version are you running? This should be fixed with #1082 .

**Comment by alblue** [comment]
Let me verify the version and if I have the fix ... will get back to you shortly.

**Comment by alblue** [comment]
Can confirm that 0.9.4 works. I had 0.9.2, which presumably had the bug fixed by #1082. Apologies for not testing the up-to-date version.
